ci/test/05_before_script.sh

```diff
@@ -33,9 +33,7 @@ if [ -z "$NO_DEPENDS" ]; then
   else
     SHELL_OPTS="CONFIG_SHELL="
   fi
-  # Temporary workaround for https://github.com/bitcoin/bitcoin/issues/16368
-  python3 -c 'import time; [print(".") or time.sleep(500) for _ in range(4)]' &
```

Comment:
I believe the issue was that this `print` did nothing, because by default `flush=False` (see https://docs.python.org/3.8/library/functions.html#print ).

Testing locally and on travis x86, it did always happen to print, just not on travis arm.